docs/resources/integration_cloudflare_account.md

```diff
@@ -33,6 +33,7 @@ resource "datadog_integration_cloudflare_account" "foo" {
 ### Optional
 
 - `email` (String) The email associated with the Cloudflare account. If an API key is provided (and not a token), this field is also required.
+- `resources` (List of String) An allowlist of resources to restrict pulling metrics for including 'web', 'dns', 'lb' (load balancer), 'worker')
```

Comment:
Are `web`, `dns`, etc. tags on the metrics? Or something else?

```suggestion
- `resources` (List of String) An allowlist of resources to restrict pulling metrics for including `web`, `dns`, `lb` (load balancer), 'worker')
```

Reply:
These are enabling which resources to collect when provisioning a cloudflare account via terraform.

Reply:
These docs are also autogenerated